SPECS-EXTENDED/python-pyroute2/python-pyroute2.spec

```diff
@@ -0,0 +1,233 @@
+%global srcname pyroute2
+
+%{!?python3_pkgversion:%global python3_pkgversion 3}
```

Comment:
We can remove this- we provide this in [`mariner-rpm-macros`](https://github.com/microsoft/CBL-Mariner/blob/5026c400b3c1d6c61a9a2b17342b3e0ef21d9e76/SPECS/mariner-rpm-macros/macros.python-srpm#L30)